src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arraySorceString = sourceString
+    .split(';')
+    .map((string) => string.trim().split(':'));
+
+  const filteredSorceString = arraySorceString
+    .filter((innerArr) => innerArr.length === 2)
+    .map((element) => element.map((item) => item.trim()));
```

Comment:
The chained methods should have the same indentation to improve readability. Align the '.map', '.filter', and '.map' methods with the '.split' method.